packages/Search/test/DTK_BoostRangeAdapters.hpp

```diff
@@ -14,20 +14,25 @@
 
 #include <DTK_DBC.hpp>
 
+#include <Kokkos_Concepts.hpp>
 #include <Kokkos_View.hpp>
 
 #include <type_traits> // is_same
 
 namespace boost
 {
+// TODO SpaceAccessibility has been promoted to Kokkos:: namespace in later
+// versions of Kokkos so evetually get rid of Impl::
```

Comment:
typo eventually

Reply:
I fixed the typo and amended the changes to the initial commit.